test/src/tests.js

```diff
@@ -1130,5 +1130,545 @@ describe('Adwords forwarder', function () {
+                },
+                {
+                    jsmap: null,
+                    map: 'other_test_consent',
+                    maptype: 'ConsentPurposes',
+                    value: 'ad_storage',
+                },
+                {
+                    jsmap: null,
+                    map: 'test_consent',
+                    maptype: 'ConsentPurposes',
+                    value: 'ad_personalization',
+                },
+            ];
+
+            beforeEach(function () {
+                window.dataLayer = [];
+                window.gtag = function () {
+                    window.dataLayer.push(arguments);
+                };
+
+                mParticle.Identity = {
+                    getCurrentUser: function () {
+                        return {
+                            getConsentState: function () {
+                                return {
+                                    getGDPRConsentState: function () {
+                                        return {
+                                            some_consent: {
+                                                Consented: false,
+                                                Timestamp: 1,
+                                                Document: 'some_consent',
+                                            },
+                                            test_consent: {
+                                                Consented: false,
+                                                Timestamp: 1,
+                                                Document: 'test_consent',
+                                            },
+                                        };
+                                    },
+                                };
+                            },
+                        };
+                    },
+                };
+            });
+
+            afterEach(function () {
+                mParticle.Identity = undefined;
+            });
+
+            it('should construct a Default Consent State Payload from Mappings', function (done) {
+                mParticle.forwarder.init(
+                    {
+                        conversionId: 'AW-123123123',
+                        enableGtag: 'True',
+                        consentMappingWeb: JSON.stringify(consentMap),
+                    },
+                    reportService.cb,
+                    true
+                );
+
+                var expectedDataLayer = [
+                    'consent',
+                    'default',
+                    {
+                        ad_user_data: 'denied',
+                        ad_personalization: 'denied',
+                    },
+                ];
+
+                // https://go.mparticle.com/work/SQDSDKS-6152
+                window.dataLayer.length.should.eql(1);
+                window.dataLayer[0][0].should.equal('consent');
+                window.dataLayer[0][1].should.equal('default');
```

Comment:
I'd argue we should use the `expectedDataLayer` values here...since they are already established, just like you are doing on the last line of this test already.
```suggestion
                window.dataLayer[0][0].should.equal(expectedDataLayer[0]);
                window.dataLayer[0][1].should.equal(expectedDataLayer[1]);
```

Reply:
Rather than add a suggested edit everywhere else in this test doc, please make that change throughout.

Reply:
As per our conversation, I think using the hardcoded values in this instance makes it more readable and reflects how consent updates and defaults are sent to Google.